styles/scss/image-gallery-no-icon.scss

```diff
@@ -287,6 +287,10 @@ $ig-transparent: rgba(0, 0, 0, 0) !default;
     margin-left: 2px;
   }
 
+  & > div {
```

Comment:
Probably worth adding a className to the `div`. Maybe call it `image-gallery-thumb-inner`

Reply:
Done